docs/tutorials/inference/latent_position_test.ipynb

```diff
@@ -50,20 +50,20 @@
     "A = sbm(csize, P)\n",
     "X = AdjacencySpectralEmbed(n_components=n_components).fit_transform(A)\n",
     "heatmap(A, title='4-block SBM adjacency matrix')\n",
-    "_ = pairplot(X, title='4-block adjacency spectral embedding')"
+    "pairplot(X, title='4-block adjacency spectral embedding')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the adjacency matrix above, there is a clearly defined block structrure corresponding to the 4 communities in the graph that we established. On the right, we see the **adjacency spectral embedding (ASE)** of this graph. $\\text{ASE}(A)$ recovers an estimate of the **latent positions** of $A$. Latent positions refer to the idea of a **random dot product graph (RDPG)** which can be modeled as follows:\n",
```

Comment:
i don't think you introduced this, but nothing is really on the right in the rendered docs. below?

Reply:
Addressed in line 60